### ISSUE TITLE
is it possible to obfuscate symfony2 projects?

### ISSUE THREAD

**Issue by ladiko** [naneau/php-obfuscator#12]
And if yes, which configuration is needed?

**Comment by naneau** [comment]
It should be possible to obfuscate any PHP code, no special configuration is needed if you point it to your `src` folder.

**Comment by gianluca78** [comment]
No, it doesn't work because Symfony2 uses annotations and php-obfuscator remove them from the source code.

**Comment by naneau** [comment]
Yes, if your code relies on annotations and other forms of meta-programming this form of obfuscation obviously won't work.

**Comment by davidecutrino** [comment]
You can use yml annotations (tested on a silex + doctrine + several symfony components project)
